backend/src/main/java/com/selflearn/backend/security/dtos/JwtResponse.java

```diff
@@ -0,0 +1,10 @@
+package com.selflearn.backend.security.dtos;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public class JwtResponse {
```

Comment:
consider using Record instead of Class for immutable class
https://www.baeldung.com/java-record-keyword